### PR TITLE
Fix: Fixing create-hint which was throwing an exception

### PR DESCRIPTION
**Pull request by vidorteg**
## Pull request checklist

Make sure you:

- [x] Signed the Contributor License Agreement (after creating PR)
- [x] Followed the [commit message guidelines](https://webhint.io/docs/contributor-guide/getting-started/pull-requests/#commit-messages)

For non-trivial changes, please make sure you also:

- [ ] Added/Updated related documentation.
- [ ] Added/Updated related tests.

## Short description of the change(s)
Fixes create-hint package which was throwing an exception when creating the hint folder. This was due some latency between the creation of the folder and copying/writing the files on it.

### CODE REVIEW

File: packages/create-hint/src/create-hint.ts [hunk 1]
```diff
@@ -190,7 +189,9 @@ class HintPackage {
  * ------------------------------------------------------------------------------
  */
 
-const mkdirpAsync = promisify(mkdirp) as (dir: string) => Promise<void>;
+const mkdirpAsync = (dir: string) => {
+    fs.mkdir(dir, {recursive: true});
```
Comment (molant) [inline packages/create-hint/src/create-hint.ts:193]: Don't you need an explicit `return` the way you've written this arrow function?

```suggestion
    return fs.mkdir(dir, {recursive: true});
```

Comment (vidorteg) [reply packages/create-hint/src/create-hint.ts]: good catch.

**Review by marty1331 — APPROVED** [review]

**Pull request merged**